packages/forms/src/directives/checkbox_value_accessor.ts

```diff
@@ -17,17 +17,28 @@ export const CHECKBOX_VALUE_ACCESSOR: any = {
 };
 
 /**
- * The accessor for writing a value and listening to changes on a checkbox input element.
+ * @description
+ * A `ControlValueAccessor` for writing a value and listening to changes on a checkbox input
+ * element.
+ * The accessor is attached to any input element that matches one of its selectors.
  *
  * @usageNotes
- * ### Example
  *
+ * ### Using an accessor with a checkbox input
```

Comment:
Hmm it's a bit misleading to say "using an accessor with a checkbox" since it implies you can use this accessor separately and that there is something you need to do other than just putting a checkbox on the page with a form directive. How about:

```suggestion
 * ### Using a checkbox with a reactive form
```